### ISSUE TITLE
Frontend: Blog page

### ISSUE THREAD

**Issue by ezrichards** [ColoradoSchoolOfMines/acm-site#18]
Explore the idea of adding a blog page. Doesn't need to be anything too crazy in terms of user review functionality, but I think a blog page would be nice.

This is low priority and should be done once the majority of the site is finished.

**Comment by ezrichards** [comment]
Closing as not planned for now.
